interface.html

```diff
@@ -20,9 +25,14 @@
     </div>
   </div>
   <div class="settings-tab state future">
-    <header>
+    <header class="header-wrapper">
```

Comment:
Why is the `header-wrapper` class necessary?

Reply:
Forget to remove this class